File: cpp/bc1043.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main()
{
	float fVal, sVal, tVal;
	
	cin >> fVal >> sVal >> tVal;
	
	bool fVer = fVal + sVal > tVal;
	bool sVer = sVal + tVal > fVal;
	bool tVer = fVal + tVal > sVal;
	
	if(fVer == sVer && sVer == tVer) cout << "Perimetro = " << fixed << setprecision(1) << fVal+sVal+tVal << endl; 
	else cout << "Area = " << fixed << setprecision(1) << ((fVal+sVal)*tVal) / 2 << endl; 
}

File: python/bc1043.py
cVal = input().split(" ")

valA = float(cVal[0])
valB = float(cVal[1])
valC = float(cVal[2])

fVer = valA + valB > valC
sVer = valB + valC > valA
tVer = valC + valA > valB

if fVer == sVer and sVer == tVer:
    per = valA + valB + valC
    print(f"Perimetro = {per:.1f}")
else:
    area = ((valA + valB) * valC) / 2
    print(f"Area = {area:.1f}")